.circleci/regenerate.py

```diff
@@ -49,6 +49,12 @@ def build_workflows(prefix="", upload=False, filter_branch=None, indentation=6):
                     ):
                         # the fields must match the build_docs "requires" dependency
                         fb = "/.*/"
+
+                    # We'll stop building Linux Wheels from CircleCI.
+                    # Keeping the Python3.8 build just for docstring_parameters_sync until this job is migrated as well.
```

Comment:
nit: Docstring Parameters Sync is also running on GHA so I think we should be good to remove the py3.8 cpu linux wheel job and the Dosctring Sync from CircleCI (if no opposition to this from the audio team :) ). If we do so, let's do it in a follow-up PR.